### ISSUE TITLE
--input-space=lsq6 ignored by MBM.py

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/pydpiper#222]
I have a set of pre-registered mouse brains in lsq6 space which I'd like to run through MBM.py using --bootstrap.

When I specify --input-space=lsq6 as per:

```
Option to specify space of input-files. Can be native (default), lsq6, lsq12.
```

It seems MBM.py ignores this completely and goes on to run an LSQ6 stage anyway.

**Comment by mcvaneede** [comment]
True... there is a quick workaround though. If you use the following two options:

```
--bootstrap --lsq6-simple
```

the code will run a 6 parameter registration, but the minctracc parameters will be set such that these stages will finish soon and should not really change your initial alignment.

**Comment by gdevenyi** [comment]
I had originally run my pipeline this way, but it ended up not working. This isn't traditional MR data, it's OPT, very high resolution and non-MR contrast so lsq6 stage doesn't work very well.

Perhaps I can create an lsq6 protocol that is mostly an "no operation". Any suggestions there?

**Comment by mcvaneede** [comment]
The command that is run looks roughly like this:

```
minctracc -clobber \
-w_translations 0.4 0.4 0.4 \
-w_rotations 0.0174533 0.0174533 0.0174533 \
-w_scales 0.02 0.02 0.02 \
-w_shear 0.02 0.02 0.02 \
-step 0.5 0.5 0.5 \
-simplex 3  \
-use_simplex \
-tol 0.0001 \
-xcorr \
-lsq6 \
-identity \
img_1_fwhm_blur.mnc \
img_2_fwhm_blur.mnc \
lsq6.xfm
```

The lsq6 protocol gives you control over the amount of blur on the input files, the step size, simplex value and w_translations. What I tried is to reduce the simplex size to virtually nothing, and it produces an identity transformation, so this should/might work (have the following in your lsq6_protocol.csv):

```
"blur";1
"step";1
"gradient";FALSE
"simplex";0.00000001
"w_translations";0.000001
```
Adjust the blur/step size based on your input files; larger step size means a faster calculation, the blur needs to fit within the field of view of your files.

**Comment by gdevenyi** [comment]
Will give that a try, thanks!

**Comment by mcvaneede** [comment]
Fixed in 2.0
